tortoise/filters.py

```diff
@@ -53,6 +53,10 @@ def not_in(field: Term, value: Any) -> Criterion:
     return field.notin(value) | field.isnull()
 
 
+def between_and(field: Term, value: Any) -> Criterion:
```

Comment:
The definition for this is PyPika is `def between(self, lower, upper):` So, could you change the type of ``value`` to ``Tuple[Any, Any]`` and then expand it to two parameters. This is just to make the code more self explanatory.